README.md

```diff
@@ -78,6 +78,17 @@ in drvs
 $ nix-build-uncached ./ci.nix
 ```
 
+### Packages with `allowSubstitutes = false`
+
+If your package set you are building has packages at the top level scope that
+have the attribute `allowSubstitutes = false;` set, than `nix-build-uncached`
+will build/download them everytime. A workaround is to use the following
+[gist](https://gist.github.com/7bc9210b1c4d0afe390f0d425b50e02f):
+
+```nix
+# TODO example
```

Comment:
@zimbatm do you have an example I can put here?

Reply:
I expect the most common one to be `pkgs.writeScriptBin` or one of the other trivial builders. All my examples are private right now.